src/components/Header/Header.tsx

```diff
@@ -141,172 +141,166 @@ const Header: FC<IHeader> = () => {
 	};
 
 	return (
-		<>
-			<StyledHeader
-				justifyContent='space-between'
-				alignItems='center'
-				theme={theme}
-				show={showHeader}
-			>
-				<Flex>
-					{isCreateRoute ? (
-						<BackBtn onClick={router.back}>
-							<Logo>
-								<Image
-									width='26px'
-									height='26px'
-									alt='Giveth logo'
-									src={`/images/back-2.svg`}
-								/>{' '}
-							</Logo>
-						</BackBtn>
-					) : (
-						<>
-							<MainLogoBtn>
-								<Link href={Routes.Home}>
-									<a>
-										<Logo>
-											<Image
-												width='48px'
-												height='48px'
-												alt='Giveth logo'
-												src={`/images/logo/logo1.png`}
-											/>
-										</Logo>
-									</a>
```

Comment:
we need the anchor tag because it's adding another internal link for us.so I think it's better to keep it.
@RamRamez